### ISSUE TITLE
Add rating to results page

### ISSUE THREAD

**Issue by cantonalex** [rubenmv/extension-goodreads-ratings-for-amazon#12]
You can add Goodreads ratings to amazon search results by using the second data-asin value.
<img width="467" alt="Screen Shot 2021-04-05 at 11 33 43 am" src="https://user-images.githubusercontent.com/18063531/113528516-22094300-9604-11eb-9485-563ae5d93514.png">
<img width="444" alt="Screen Shot 2021-04-05 at 11 33 19 am" src="https://user-images.githubusercontent.com/18063531/113528526-2b92ab00-9604-11eb-8bb0-fc90b9ea4083.png">

**Comment by jpinkham** [comment]
Yes, please.  Having ratings available on search results pages would be magnificent
